pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools >= 61", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "league-rpc-linux"
+dynamic = ["version"]
+requires-python = ">= 3.8"
+dependencies = [
```

Comment:
I believe we will need to add the new dependencies that got added since this PR was initially made.

I think that would only be lcu-driver

Reply:
These don't really matter unless you install with `pip`. Anyway updated it.